test/widgets/test_widget_init_only.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+
+import libqtile.widget as widgets
+from libqtile.widget.crashme import _CrashMe
+
+# This file runs a very simple test to check that widgets can be initialised
+# and that keyword arguments are added to default values.
+#
+# This test is not meant to replace any widget specific tests but should catch
+# any mistakes that inadvertently breakag widgets.
+#
+# By default, the test runs on every widget that is listed in __init__.py
+# This is done by building a list called `parameters` which contains a tuple of
+# (widget class, kwargs).
+#
+# Adjustments to the tests can be made below.
+
+# Some widgets require certain parameters to be set when initialising.
+# Widgets listed here will replace the default values
+overrides = [
+    (
+        widgets.BitcoinTicker,
+        {"currency": "USD"}  # Set the currency to prevent locale lookup
+    ),
+    (
+        widgets.Maildir,  # Sub-folders needs to defined in correct format
+        {"sub_folders": [{"path": "INBOX", "label": "Home mail"}]}
```

Comment:
seems like you can just move this to Maildir, everyone is defining their own already anyway

Reply:
Yeah - that was my feeling. Will do that in a separate PR.

Reply:
#2227